Gathered fixes from abandoned PRs. (#6762)
magento/devdocs: src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md

   }
   ```

> 1. You must edit the `app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml` file to declare your fonts.  Otherwise Magento 2.3 will not be able to create the `static/web/fonts/` directory. See `app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml`.

Reviewer: ```suggestion
1. You must edit the `app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml` file to declare your fonts; otherwise, Magento 2.3 is not able to create the `static/web/fonts/` directory. See `app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml`.
```

Reviewer 2: I disagree with the suggestion to use a semicolon to convert two sentences to one. I would change "will not be able to" to "cannot".

Reviewer: No big deal. You don't have to join the sentences. But you SHOULD get rid of the be verb.
